### ISSUE TITLE
Update Performance documentation

### ISSUE THREAD

**Issue by annashamray** [open-zaak/open-zaak#1785]
### Thema / Theme

Other

### Omschrijving / Description

This page https://open-zaak.readthedocs.io/en/latest/development/performance/index.html needs some update:

* add system requirements of the test server
* add performance test results for 8 users and OZ 1.14.0 version

**Comment by annashamray** [comment]
I've tested OZ 1.14.0 (fuzzy on) with different number of users. 
Do you want me to include some of it into the documentation? @alextreme @joeribekker 

```
=========== ========= ======================
Users       Role      Median time (in ms)
=========== ========= ======================
1           Superuser 640
1           Regular   690
2           Superuser 690
2           Regular   810
4           Superuser 1600
4           Regular   1700
8           Superuser 2000
8           Regular   2600
16          Superuser 2600
16          Regular   3400
=========== ========= ======================
```

**Comment by alextreme** [comment]
@annashamray I don't think adding these tests to the documentation is worthwhile, but I do think Joeri will find these results interesting in that the performance degrades more or less linearly with more simultaneous users. A more common pattern is seeing that the performance is okay up until a certain number of users, and degrades quickly above that point.

It shows in my eyes that there is a bottleneck when scaling >1 (and I suspect that this is due to the DB/table-locking?)

**Comment by annashamray** [comment]
@alextreme Yes, the linear degradation was surprising. Also for 2 users the CPU usage was around 17%, so it's not like it was all utilized. For now OZ is deployed with 1 replica, 4 processes and 4 threads. Maybe increasing the amount of replicas could affect the result and CPU usage, but it's just my theory

**Comment by annashamray** [comment]
@alextreme We don't do any write operations to DB here and REST API doesn't use sessions, so I can't see where the DB locking can happen -_-

**Comment by alextreme** [comment]
> @alextreme We don't do any write operations to DB here and REST API doesn't use sessions, so I can't see where the DB locking can happen -_-

Neither can I. I attempted a few changes in the Postgresql configuration locally (max_parallel_workers for instance) however this didn't affect performance based on my tests.

I suggest to discuss the results with Joeri before proceeding

**Comment by joeribekker** [comment]
* [ ] Check CPU with 4 users (if this hits 100%, lets discuss later)
* [ ] Try with 2 replicas, instead of 1, 4 users, the million cases
* [ ] Run performance test with 4 users on 1000 cases instead 1 million (less data)
